examples/ima.amp.html

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
+<html ⚡>
+<head>
+    <meta charset="utf-8">
+    <title>IMA examples</title>
+    <link rel="canonical" href="amps.html" >
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    <link href='https://fonts.googleapis.com/css?family=Questrial' rel='stylesheet' type='text/css'>
+    <link href="//vjs.zencdn.net/5.3/video-js.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="http://gvabox.com/sbusolits/amp/css/videojs.ads.css" />
```

Comment:
@shawnbuso this is invalid in amp. You are not allowed to load external CSS. This is probably why your CSS is not working.

AMP [only allows embedded custom CSS in the header](https://github.com/ampproject/amphtml/blob/master/spec/amp-html-format.md#author-stylesheets) with `<style amp-custom>` tag. 

To see these validation errors. Try navigating to and check the console:
http://localhost:8000/examples.build/ima.amp.max.html#development=1